src/rageshake/rageshake.ts

```diff
@@ -444,6 +444,11 @@ export function init(setUpPersistence = true): Promise<void> {
     global.mx_rage_logger = new ConsoleLogger();
     global.mx_rage_logger.monkeyPatch(window.console);
 
+    // log unhandled rejections in the rageshake
+    window.addEventListener("unhandledrejection", (event) => {
+        logger.error(`Unhandled promise rejection: ${event.reason}`);
+    });
```

Comment:
question: does the unhandled rejection still get logged in the console the same way as it was before? Or is it now formatted differently? Or is it logged twice?

Reply:
It gets logged twice (see the screenshot in https://github.com/matrix-org/matrix-react-sdk/pull/12028#issuecomment-1853111552).

I guess that if I use `global.mx_rage_logger.error(...)`, then it will only log once.

Reply:
then let's do that :)

Reply:
done